.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
     push:
         branches:
             - main
-    pull_request: # Runs on pull requests to any branch
+    pull_request:
```

Comment:
Comment not necessary?

Reply:
Well... here's the thing, it is shocking to me how many comments AIs will just rip out. I catch most of them but some of the slip through. This is one of those. However, this comment is not really necessary. That is what putting pull_request there mean. And we don't have a corresponding comment on `push:` that says "Runs on pushes to main branch." Anyone that knows the github workflow yml should know that is what it means. And we don't put a comment on a for loop that says "repeats the below code 10 times" because anyone that knows javascript will know that's exactly what a for loop does. So yeah, it's not really necessary